test/fixtures/spina/admin/journal/affiliations.yml

```diff
@@ -0,0 +1,20 @@
+marcus:
+  institution: rockbottom
+  author: marcus
+  first_name: Marcus
+  surname: Atherton
+  status: 1
+
+toope:
+  institution: miami
+  author: marcus
+  first_name: Stephen
+  surname: Toope
+  status: 0
```

Comment:
😂

Reply:
hahaha... manually coming up with fixture names is a bit of a psych experiment